src/sdks/tableau/types/pulse.ts

```diff
@@ -117,6 +123,225 @@ export const pulseMetricSubscriptionSchema = z.object({
   metric_id: z.string(),
 });
 
+// prettier-ignore
```

Comment:
Please remove these `// prettier-ignore` comments. If you have the Prettier extension installed and Format on Save enabled in VS Code settings, it should auto replace the `"` with `'`' characters.